client.py

```diff
@@ -87,12 +89,11 @@ def main():
                 if ('cpu_only' in config_cluster and config_cluster['cpu_only']
                     and s["gpus"] != 0):
                     continue
-                if (i >= config_cluster["limit_per_submit"]
-                    or i + glideins_running >= config_cluster["max_total_jobs"]):
+                if (i >= limit):
```

Comment:
Move the limit condition to the top of the loop.  and remove the `()` now that it's only a single line.